netbox/netbox/filtersets.py

```diff
@@ -29,6 +29,13 @@
     'OrganizationalModelFilterSet',
 )
 
+STANDARD_LOOKUPS = (
+    'exact',
+    'iexact',
+    'in',
+    'contains',
+)
```

Comment:
I was originally going to suggest that maybe this should be a set instead of a tuple, since it only gets instantiated once but accessed a lot later (on the order of ~2500 times within 10 list view requests). However, some quick measurements showed that with only 4 items, the tuple is actually a little faster for lookup than the set.